example/src/hooks/index.tsx

```diff
@@ -28,8 +28,8 @@ export const useSessionConfiguration = (): UseSessionConfigurationResponse => {
     ] = useState<SessionConfiguration>({
         allowsCellularAccess: true,
         waitsForConnectivity: false,
-        timeoutIntervalForRequest: 30000,
-        timeoutIntervalForResource: 30000,
+        timeoutIntervalForRequest: 60000,
+        timeoutIntervalForResource: 604800000, // 7 days
```

Comment:
should we keep it undefined so that the OS applies the default values instead?

Reply:
This is also used on android apparently (`android/src/main/java/com/mattermost/networkclient/NetworkClientBase.kt`). So... that part will also get the default one.

So... I guess yes, that is the better approach.